library/elixir

```diff
@@ -1,19 +1,34 @@
-# this file is generated via https://github.com/c0b/docker-elixir/blob/e2d20313fefcd13b8d3b1bcd94df60bd4f02a0b3/generate-stackbrew-library.sh
+# this file is generated via https://github.com/erlef/docker-elixir/blob/a6086e61c4d193a569e046d57f212872345839e3/generate-stackbrew-library.sh
 
 Maintainers: . <c0b@users.noreply.github.com> (@c0b)
```

Comment:
Should this be updated as well?

Reply:
I think we should respect him and his work.

Reply:
Of course. I meant, add a new maintainer to the list.

Reply:
I don't want to add myself.

Reply:
If @c0b is no longer maintaining this image, it is not appropriate for @c0b to be the only human listed (or even to be listed at all, really) -- we use this list for pinging maintainers on related issues (docs changes, base image changes, etc), so we really need some humans here.  The format is either `Name <email> (@github)` or `Name (@github)` so it doesn't really need to be anything elaborate.

I don't have a problem with leaving @c0b listed, but I really wish we could get some kind of reply that this is OK with them. :disappointed:  (I hope they're doing ok!)

Reply:
@getong since you don't want to be added I can be used:`Tristan Sloughter t@crashfast.com (@tsloughter)` -- we should also put `Erlang Ecosystem Foundation Build and Packaging Working Group https://github.com/erlef/eef-build-and-packaging-wg` since that working group is really who will be maintaining it.

Reply:
Add Tristan Sloughter as the maintainer, the pr now is ready to be merged.